src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
My second suggestion addresses the concern that the element is an upgraded custom element of the expected tagname. We could add:

```javascript
assert.true(Polymer.telemetry.registrations.indexOf(element.localName) !== -1);
```

To also assert that the Polymer element class was finalized without issue (all the properties, observers, and templates have been associated to the class)